### PR TITLE
3 cleanup commits

### PR DESCRIPTION
**Pull request by kevmoo**
Will be landed independently

### CODE REVIEW

**Comment by kevmoo** [comment]
Feel free to **rebase** and merge and delete the branch if you're happy

File: pubspec.yaml [hunk 1]
```diff
@@ -3,10 +3,10 @@
 # BSD-style license that can be found in the LICENSE file.
 
 name: usage
-version: 3.4.2-dev
-description: A Google Analytics wrapper for both command-line, web, and Flutter apps.
+version: 3.4.2
+description: >-
+  A Google Analytics wrapper for both command-line, web, and Flutter apps.
```
Comment (natebosch) [inline pubspec.yaml:8]: The "both" is awkward here. Should we rephrase it?

File: readme.md [hunk 2]
```diff
@@ -112,11 +110,6 @@ For more information, please see the Google Analytics Measurement Protocol
 
 Tests can be run using `pub run test`.
 
-## Issues and bugs
-
-Please file reports on the
-[GitHub Issue Tracker](https://github.com/dart-lang/usage/issues).
-
 ## License
```
Comment (natebosch) [inline readme.md:113]: Do any other README we own bother to mention the license specifically?

File: pubspec.yaml (hunk 1 above)
Comment (kevmoo) [reply pubspec.yaml]: ack

File: readme.md (hunk 2 above)
Comment (kevmoo) [reply readme.md]: ack

**Comment by natebosch** [comment]
Hopefully removing "both" didn't make the description too short 😄

**Pull request merged**